src/schema/metadata/MultibandAccelerationFactor.yaml

```diff
@@ -0,0 +1,5 @@
+---
+name: MultibandAccelerationFactor
+description: |
+  The multiband factor, for multiband acquisitions.
+type: integer
```

Comment:
The specification table lists this as a `number`. Should it be one?

Reply:
JSON does not distinguish between integers and floats, and just has the concept of "number". If we want to describe floats and ints, we can, but maybe reserve it for a later PR?

Reply:
but JSON-Schema has a concept for integer: https://json-schema.org/understanding-json-schema/reference/numeric.html#integer (with a big caveat that's described there)

Reply:
Okay, I'm fine with it in general, but I still think it should be a different PR so that the decision to do this doesn't get buried in the comment thread but instead gets a full changelog entry to itself.

Reply:
I can go through the examples and revert the types but add comments. Some of these changes are the product of mismatch between the spec and the validator, but I agree that that doesn't have to be dealt with in this PR.